ossec_security/assets/dashboards/ossec_sshd.json

```diff
@@ -0,0 +1,739 @@
+{
+    "title": "OSSEC - SSHD",
+    "description": "-   This dashboard offers comprehensive insights into SSHD events and activities conducted through SSH-like connection attempts, authentication failure.",
+    "widgets": [
+        {
+            "id": 6582953341186730,
+            "definition": {
+                "type": "image",
+                "url": "https://www.ossec.net/wp-content/uploads/2019/01/ossec-300x113.png",
+                "url_dark_theme": "https://www.ossec.net/wp-content/uploads/2019/01/ossec_logo_2-300x113.png",
+                "sizing": "contain",
+                "has_background": false,
+                "has_border": false,
+                "vertical_align": "center",
+                "horizontal_align": "center"
+            },
+            "layout": {
+                "x": 0,
+                "y": 0,
+                "width": 3,
+                "height": 2
+            }
+        },
+        {
+            "id": 3742523071554850,
+            "definition": {
+                "type": "note",
+                "content": "## Overview\n- This dashboard offers comprehensive insights into SSHD events and activities conducted through SSH-like connection attempts, authentication failure.\n",
```

Comment:
```suggestion
                "content": "## Overview\n- This dashboard offers comprehensive insights into SSHD events and activities conducted through SSH-like connection attempts, such as authentication failure.\n",
```

Reply:
Connection attempts and authentication failures are distinct types of events by SSHD.